internal/eventstore/models/search_query.go

```diff
@@ -0,0 +1,74 @@
+package models
+
+import "github.com/caos/zitadel/internal/errors"
+
+type SearchQuery struct {
+	Limit   uint64
+	Desc    bool
+	Filters []*Filter
+}
+
+func NewSearchQuery() *SearchQuery {
+	return &SearchQuery{
+		Filters: make([]*Filter, 0, 4),
```

Comment:
why you know you have 4 filters?

Reply:
I set 4 as capacity for all implemented filters